force-app/main/default/pages/STG_PanelHouseholds.page

```diff
@@ -161,15 +161,15 @@
                     <apex:outputText value="{!stgService.stgHN.Formal_Greeting_Format__c}" rendered="{!isReadOnlyMode}" styleClass="slds-form-element__static" />
                     <apex:outputPanel rendered="{!isEditMode}" >
                         <apex:selectList value="{!stgService.stgHN.Formal_Greeting_Format__c}" html-aria-describedby="{!$Component.slstrFormatFGHelp}" styleClass="slds-select"
-                            onchange="setNameSpec(this, 'txtFormatFG', '{!$Label.stgLabelExamplesFGFormat}');"
-                            onfocus="setNameSpec(this, 'txtFormatFG', '{!$Label.stgLabelExamplesFGFormat}');"
+                            onchange="setNameSpec(this, 'txtFormatFG', '{!JSENCODE($Label.stgLabelExamplesFGFormat)}');"
+                            onfocus="setNameSpec(this, 'txtFormatFG', '{!JSENCODE($Label.stgLabelExamplesFGFormat)}');"
                             multiselect="false" size="1" rendered="{!isEditMode}" id="slstrFormatFG" >
                                 <apex:selectOptions value="{!listSOFormalGreetingFormats}"/>
                         </apex:selectList>
-                        <apex:outputLabel value="{!$Label.stgLabelOtherFormalGreetingFormat}" styleClass="slds-assistive-text" for="txtFormatFG" />
+                        <apex:outputLabel value="{!JSENCODE($Label.stgLabelOtherFormalGreetingFormat)}" styleClass="slds-assistive-text" for="txtFormatFG" />
```

Comment:
Any particular reason the custom label in this outputLabel component is also being jsencoded? I don't think it's necessary and at the same time doesn't harm anything. Just want to make sure I'm not missing something.

Reply:
Good catch, yeah, was trying to get all of them where the value is interpreted as JS.  The output label here doesn't need it.  Removed.